packages/app/src/server/routes/api/cortex/index.ts

```diff
@@ -31,6 +32,34 @@ export const cortexProxy = httpProxy.createProxyServer({ ignorePath: true });
  */
 cortexProxy.on("proxyReq", onProxyReq);
 
+cortexProxy.on("proxyRes", (proxyRes, req, res) => {
+  const isHTMLResponse = proxyRes.headers["content-type"]?.includes("text/html")
+  let data: any = undefined;
+  const body: Array<any> = [];
+  res.writeHead(proxyRes.statusCode!, proxyRes.headers);
+  proxyRes.on("data", function (chunk) {
+    if (!isHTMLResponse) {
+      res.write(chunk)
+    }
+    body.push(chunk)
```

Comment:
If the response is HTML, it's not sent to the client, but rather collected in `body`, [to be later modified](https://github.com/opstrace/opstrace/pull/900/files#diff-4ba3f8112ae1c4d28b5397690668351095b7568556ad0bce0d6e2bf0b067d4dbR47-R57).